mappings/net/minecraft/entity/passive/CatEntity.mapping

```diff
@@ -1,13 +1,13 @@
 CLASS net/minecraft/unmapped/C_rxtmregr net/minecraft/entity/passive/CatEntity
-	COMMENT Meow.
+	COMMENT mrrp nya
```

Comment:
lmao

Reply:
nooooo I was trying to slip it in unnoticed